libraries/AP_Mount/AP_Mount.h

```diff
@@ -19,6 +19,19 @@
 ************************************************************/
 #pragma once
 
+#include <AP_HAL/AP_HAL.h>
+#include <AP_AHRS/AP_AHRS.h>
+
+#ifndef HAL_MOUNT_ENABLED
+#define HAL_MOUNT_ENABLED !HAL_MINIMIZE_FEATURES
+#endif
+
+#ifndef HAL_SOLO_GIMBAL_ENABLED
+#define HAL_SOLO_GIMBAL_ENABLED HAL_MOUNT_ENABLED && AP_AHRS_NAVEKF_AVAILABLE
```

Comment:
IMHO solo_gimbal should be disable by default and we should just make a build for solo

Reply:
I tend to agree